set.mm

```diff
@@ -511744,20 +511744,69 @@ have become an indirect lemma of the theorem in question (i.e. a lemma
   $}
 
   ${
-    $d x y z $.  $d w x z $.
+    $d w x z $.  $d v x y z $.
     $( If the Axiom of Infinity is denied, then the Axiom of Power Sets becomes
```

Comment:
```suggestion
    $( If the Axiom of Infinity is negated, then the Axiom of Power Sets becomes
```
Same below. Less ambiguous, since "denied" could mean that we simply forbid its use, instead of assuming its negation.

Reply:
I updated the descriptions for my theorems. Variants of "deny" are used in this way throughout set.mm, but I'm considering those cases to be beyond the scope of this change.